cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2959,6 +2972,11 @@ fn aarch64_map_regs<RUM: RegUsageMapper>(inst: &mut Inst, mapper: &RUM) {
         }
         &mut Inst::VirtualSPOffsetAdj { .. } => {}
         &mut Inst::EmitIsland { .. } => {}
+
+        &mut Inst::ElfTlsGetAddr { .. } => {
+            // TODO: This is probably wrong
```

Comment:
This is correct. All registers are fixed, so marking them as clobbered in `aarch64_get_regs` is enough. There are no registers that need to be remapped.